lib/core/versions/latest/UpdateOperation.ts

```diff
@@ -112,9 +113,12 @@ export default class UpdateOperation implements OperationModel {
       if (operationObject.type !== OperationType.Update) {
         throw new SidetreeError(ErrorCode.UpdateOperationTypeIncorrect);
       }
-
-      encodedDelta = operationObject.delta;
-      delta = await Operation.parseDelta(encodedDelta);
+      Operation.validateDelta(operationObject.delta);
+      delta = {
+        patches: operationObject.delta.patches,
+        updateCommitment: operationObject.delta.update_commitment
+      };
+      encodedDelta = Encoder.encode(JsonCanonicalizer.canonicalizeAsBuffer(operationObject.delta));
```

Comment:
Design: I think/hoped that all of `encodedDelta` (and `encodedSuffixData`) can now be removed with SIP2. The point of keeping these "encoded" form was really just for hash checks, but we no longer need them (with exception to perhaps the create operation where old long-form DID still needs this old way of hash validation).